### DISCUSSION TITLE
5.0.5.0

### DISCUSSION THREAD

**Discussion by mgth** [mgth/LittleBigMouse#293] [Announcements]
- Fix strait algo did not work for bottom crossing.
- Add bug report on crash (please copy details in bug report).
- Disable VCP contrôle until I can make it safer (will be optional/experimental in next version).
- Fix some "move around" problems in layout editing.

<hr /><em>This discussion was created from the release <a href='https://github.com/mgth/LittleBigMouse/releases/tag/v5.0-beta5'>5.0.5.0</a>.</em>

**Comment by ciber85** [comment]
Hi. Still not running on my W11 PC

**Comment by Nameless-sparrow** [comment]
It need windows runtime but when i finished install runtime V8.0,it still ask me to install runtime,maybe its cant detect the runtime

**Reply by FliesEyes** [reply]
Did you get this run on your system. Doesn't seem to want load yet.  I must be missing something.

**Comment by KevSobz** [comment]
none of the versions i have downloaded works. it opens the run as administrator window, after clicking yes nothing shows up. some versions usually show up in task manager but thats pretty much it.

**Comment by smcd0** [comment]
I've downloaded about five of the most recent versions on various different drives, ran with all compatability settings enabled, restarted computer a number of times, etc. Nothing works on my w11 PC. As of writing, I'm using the latest version of windows 11. One interesting thing is that my original installation actually did boot on this version of Windows 11, but upon removing it and reinstalling it, it ceased to work.

**Comment by Artiom97es** [comment]
all of you, check all kind of runtimes... Works fine on all my win11(tested on 7 of them)

**Comment by MakcYan** [comment]
Hello. If two horizontal monitors are located in the system (the first is 27' FullHD, the second is 27' 2K) and when the "Detach" of the second monitor is deactivated, the "Horizontal loop" will not work. The vertical loop works at the same time.

**Comment by F1reD1amond** [comment]
Been following some trouble shooting procedures by uninstalling and deleting the files completely, and installing an earlier version, and the latest version that worked on my win 11 pc is the 5.0.1.0, any version later than that won't work.

**Reply by BruhIdkTbh** [reply]
yes 5.0.10 worls thank u

**Comment by PaulSorauer-IPG** [comment]
I've tried installing various versions from 5.0.4.1 up (I want the 'same monitor' fix) but none of them run on my Win 11. I get a blank icon in my system tray (a blank area with a rollover on mouse hover but no icon visible), as well as a LBM process running in Task Manager.

I'm on Win 11 23H2 (OS build 22631.3227) and have .NET 8 installed.

As an FYI, I also can't get any versions to compile - there's something strange that happens with the submodules in git and just opening the repos, HLab Core wants to remove 100+ files and HLab Avalonia wants to remove 500+ files. I reset all changes and allowed the main repo to re-sync with the submodules, but when opening in VS, nuget failed to find any of the Avalonia project refs. I gave up trying to get the source to compile at that point (like I've given up each of the roughly 5 times I've previously tried to compile the code in the last 5-8 years).

**Reply by FliesEyes** [reply]
I am getting similar results. I was working on my previous PC but struggling to get to work this time around.

**Reply by vikas65** [reply]
Did you find the solution for it? I am facing exactly the same issue. After installing the 5.0.5.0v LBM and windows runtime, not able to open the LBM app.

**Reply by PaulSorauer-IPG** [reply]
Sadly no 🥲

**Reply by vikas65** [reply]
Any previous working LBM version?

**Reply by PaulSorauer-IPG** [reply]
Yes, the 4.2.7 version from 2019:

https://github.com/mgth/LittleBigMouse/releases/tag/4.2.7124.42685

It's functional, but has a problem identifying monitors if you have multiple monitors of the same brand / make.

**Comment by Moosespank** [comment]
![image](https://github.com/mgth/LittleBigMouse/assets/160975421/5816d84e-0d19-41d8-9da4-5ef3ac9d9585)
I am having this issue where I cant see any monitor. Has anyone run into this issue?
